### PR TITLE
Fix issues suggested by codacy.

### PR DESCRIPTION
**Pull request by lgarithm**

### CODE REVIEW

**Comment by lgarithm** [comment]
@luomai @zsdonghao could you help setup the webhook for codacy, instruction is [here](https://app.codacy.com/app/tensorlayer/tensorlayer/settings/integrations).

**Comment by lgarithm** [comment]
@luomai @zsdonghao when you finish setup the webhook, please reply to this thread, to trigger a comment event.

**Comment by lgarithm** [comment]
still not reviewed by codacy, don't merge.

**Comment by luomai** [comment]
https://app.codacy.com/app/tensorlayer/tensorlayer/files?bid=6613409

**Comment by lgarithm** [comment]
@luomai you can also filter issus by pattern [here](https://app.codacy.com/app/tensorlayer/tensorlayer/issues/index?bid=6613409).

File: example/tutorial_mnist.py [hunk 1]
```diff
@@ -204,7 +203,7 @@ def main_test_denoise_AE(model='relu'):
 
     # placeholder
     x = tf.placeholder(tf.float32, shape=[None, 784], name='x')
-    y_ = tf.placeholder(
+    tf.placeholder(
```
Comment (lgarithm) [inline example/tutorial_mnist.py]: The whole statement should be removed, not just `y_`
```
    y_ = tf.placeholder(
        tf.int64, shape=[
            None,
        ], name='y_')
```

File: example/tutorial_mnist.py [hunk 2]
```diff
@@ -301,7 +296,7 @@ def main_test_stacked_denoise_AE(model='relu'):
 
     # Define fine-tune process
     y = network.outputs
-    y_op = tf.argmax(tf.nn.softmax(y), 1)
+    tf.argmax(tf.nn.softmax(y), 1)
```
Comment (lgarithm) [inline example/tutorial_mnist.py]: `tf.argmax(tf.nn.softmax(y), 1) `
should also be removed

File: example/tutorial_cifar10_tfrecord.py [hunk 3]
```diff
@@ -71,7 +71,7 @@ def data_to_tfrecord(images, labels, filename):
         print("%s exists" % filename)
         return
     print("Converting data into %s ..." % filename)
-    cwd = os.getcwd()
+    # cwd = os.getcwd()
```
Comment (lgarithm) [inline example/tutorial_cifar10_tfrecord.py]: just delete this line if now used.

File: example/tutorial_cifar10_tfrecord.py [hunk 4]
```diff
@@ -10,43 +10,43 @@
 import tensorlayer as tl
 from PIL import Image
 from tensorlayer.layers import *
-"""Reimplementation of the TensorFlow official CIFAR-10 CNN tutorials:
```
Comment (luomai) [inline example/tutorial_cifar10_tfrecord.py]: Put before imports @zsdonghao

File: example/tutorial_frozenlake_q_table.py [hunk 5]
```diff
@@ -1,18 +1,17 @@
...
-Q(S, A) <- Q(S, A) + alpha * (R + lambda * Q(newS, newA) - Q(S, A))
-
-See David Silver RL Tutorial Lecture 5 - Q-Learning for more details.
-
-For Q-Network, see tutorial_frozenlake_q_network.py
 
-EN: https://medium.com/emergent-future/simple-reinforcement-learning-with-tensorflow-part-0-q-learning-with-tables-and-neural-networks-d195264329d0#.5m3361vlw
-CN: https://zhuanlan.zhihu.com/p/25710327
-"""
+# Q-Table learning algorithm, non deep learning - TD Learning, Off-Policy, e-Greedy Exploration
```
Comment (luomai) [inline example/tutorial_frozenlake_q_table.py]: @zsdonghao  put before imports

File: example/tutorial_inceptionV3_tfslim.py [hunk 6]
```diff
@@ -12,35 +12,34 @@
 import skimage.transform
 import tensorflow as tf
 import tensorlayer as tl
-from scipy.misc import imread, imresize
-from tensorflow.contrib.slim.python.slim.nets.alexnet import alexnet_v2
-from tensorflow.contrib.slim.python.slim.nets.inception_v3 import (inception_v3, inception_v3_arg_scope, inception_v3_base)
+# from scipy.misc import imread, imresize
```
Comment (luomai) [inline example/tutorial_inceptionV3_tfslim.py]: can we remove this?
